specification/mysql/resource-manager/readme.md

```diff
@@ -82,12 +82,31 @@ input-file:
 
 These settings apply only when `--tag=package-2020-01-01-privatepreview` is specified on the command line.
 
-
 ``` yaml $(tag) == 'package-2020-01-01-privatepreview'
 input-file:
 - Microsoft.DBforMySQL/preview/2020-01-01-privatepreview/DataEncryptionKeys.json
 ```
 
+
+### Tag: package-2020-01-01-preview
+
+These settings apply only when `--tag=package-2020-01-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-01-01-preview'
```

Comment:
see the comments below as well - tags are intended to represent a snapshot of your entire API, not just  the single api you are adding - see https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md as an example